help/md/dnCBDSP.md

```diff
@@ -1,9 +1,39 @@
 ## name
 dnCBDSP
 ## title
+Conditional birth-death shift process
 ## description
+Simulates a tree under a birth-death process with shift in birth and death rates.
 ## details
+This function is a flexible simulator that can be used for several phylogenetic models.
+dnCBDSP accepts several arguments to simulate a birth-death process:
```

Comment:
I think specifying the argument meanings in a list here is redundant, since typing `?dnCBDSP` will list them and their descriptions anyway.

Can you instead describe the process?  It sounds like the birth and death rates change over time.  How are they initialized?  How do they change?